index.html

```diff
@@ -8,7 +8,7 @@
     <meta name="description" content="">
     <meta name="author" content="">
 
-    <title>Gaulab</title>
+    <title>AppScaleTech</title>
```

Comment:
also please put space between Appscale and Tech, AppScaleTech is only a repository name